pkgs/by-name/pi/pid-fan-controller/package.nix

```
@@ -0,0 +1,29 @@
+{
+  rustPlatform,
+  fetchFromGitHub,
+  lib,
+}:
+let
+  version = "0.1.1";
+in
+rustPlatform.buildRustPackage {
+  pname = "pid-fan-controller";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "zimward";
+    repo = "pid-fan-controller";
+    rev = version;
+    hash = "sha256-ALR9Qa0AhcGyc3+7x5CEG/72+bJzhaEoIvQNL+QjldY=";
+  };
+  cargoHash = "sha256-u1Y1k1I5gRzpDHhRJZCCtMTwAvtCaIy3uXQTvmtEx5w=";
```

```suggestion
  cargoHash = "sha256-Y57VSheI94b43SwNCDdFvcNxzkA16KObBvzZ6ywYAyU=";
```
The default cargo fetcher was updated a while ago, all hashes had to be replaced.